code/datums/outfits/outfit_admin.dm

```diff
@@ -1354,3 +1354,42 @@
 	if(istype(I))
 		apply_to_card(I, H, list(ACCESS_CLOWN), "Emergency Response Clown")
 	H.sec_hud_set_ID()
+
+/datum/outfit/admin/observer
+	name = "Observer"
+
+	uniform = /obj/item/clothing/under/costume/tourist_suit
+	back = /obj/item/storage/backpack/satchel
+	shoes = /obj/item/clothing/shoes/black
+	backpack_contents = list(
+		/obj/item/storage/box/survival = 1,
+		/obj/item/implanter/dust = 1
```

Comment:
Maybe this implant should be automatically implanted?

Reply:
I thought about this, but sometimes you wanna test stuff that might have you die. Auto-dusting when you die means you'd have to spawn people in every time. Sometimes we test shit and just Healing Staff people back up.